S/Sokol/build_tarballs.jl

```diff
@@ -0,0 +1,71 @@
+using BinaryBuilder
+import Pkg: PackageSpec
+
+name = "Sokol"
+version = v"2025.28.1"  # Use the commit date or tag of Sokol you're targeting
+
+# Use the latest commit or a specific tag from the Sokol repository
+sources = [
+    GitSource("https://github.com/floooh/sokol.git", "db9ebdf24243572c190affde269b92725942ddd0"),
+    DirectorySource("./bundled"),
+]
+
+# Build script
+script = raw"""
+cd $WORKSPACE/srcdir/sokol*
+export CFLAGS="-I${includedir}"
+cp ${WORKSPACE}/srcdir/files/CMakeLists.txt ./CMakeLists.txt
+cp ${WORKSPACE}/srcdir/files/sokol.c ./sokol.c
+
+if [[ ${target} == aarch64-apple-* ]]; then
+    # `libclang_rt.osx.a` from LLVM compiler-rt.
+    FLAGS+=(-DCMAKE_SHARED_LINKER_FLAGS="-L${libdir}/darwin -lclang_rt.osx"
+            -DCMAKE_EXE_LINKER_FLAGS="-L${libdir}/darwin -lclang_rt.osx"
+            -DUSE_METAL=ON
+            )
+fi
+
+cmake -B build \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    "${FLAGS[@]}"
+
+cmake --build build
+
+install -Dm 755 "build/libsokol.${dlext}" "${libdir}/libsokol.${dlext}"
+install -Dm 755 "sokol_gfx.h" "${includedir}/sokol_gfx.h"
+install -Dm 755 "sokol_app.h" "${includedir}/sokol_app.h"
+install -Dm 755 "sokol_audio.h" "${includedir}/sokol_audio.h"
+install -Dm 755 "sokol_time.h" "${includedir}/sokol_time.h"
+install -Dm 755 "sokol_glue.h" "${includedir}/sokol_glue.h"
+install -Dm 755 "sokol_log.h" "${includedir}/sokol_log.h"
+install -Dm 755 "sokol_args.h" "${includedir}/sokol_args.h"
+install -Dm 755 "sokol_fetch.h" "${includedir}/sokol_fetch.h"
```

Comment:
1. header files aren't executable
2. can't just iterate over them?
```suggestion
for file in sokol_*.h; do
    install -Dvm 644 "${file}" -t "${includedir}"
done
```